src/rules/consistent-test-it.ts

```diff
@@ -56,7 +71,7 @@ export default createRule<
     let describeNestingLevel = 0;
 
     return {
-      CallExpression(node) {
+      CallExpression(node: TSESTree.CallExpression) {
```

Comment:
I added this because of WebStorm - for some reason it doesn't quite infer the `node` type properly, so won't show guard changes without the explicit type.

I've been meaning to make an issue on YouTrack for it, but haven't gotten around to doing so yet.

Happy to remove, but we're not overly consistent, and the selector ones are required (i.e when doing things like `'CallExpression:exit'`)